python/lsst/ts/idl/enums/MTMount.py

```diff
@@ -137,8 +139,18 @@ class System(enum.IntEnum):
     OIL_SUPPLY_SYSTEM = 9
     AZIMUTH_DRIVES_THERMAL = 10
     ELEVATION_DRIVES_THERMAL = 11
-    AZ0101_CABINET_THERMAL = 12
-    MODBUS_TEMPERATURE_CONTROLLERS = 13
+    CABINET_0101_THERMAL = 12
+    AZ0101_CABINET_THERMAL = 12  # deprecated alias
+    AUXILIARY_CABINETS_THERMAL = 13
+    MODBUS_TEMPERATURE_CONTROLLERS = 13  # deprecated alias
     MAIN_CABINET_THERMAL = 14
     MAIN_AXES_POWER_SUPPLY = 15
     TOP_END_CHILLER = 16
+
+
+class ThermalCommandState(enum.IntEnum):
+    """State for the setThermal command."""
+
+    NO_CHANGE = 0
+    OFF = 1
+    ON = 2
```

Comment:
As in the ts_xml PR, shouldn't there be a `FAULT` value here as well?

Reply:
No. See my response in your ts_xml review.